OpenRA.Mods.Common/Traits/Render/WithMakeAnimation.cs

```diff
@@ -17,6 +17,7 @@
 
 namespace OpenRA.Mods.Common.Traits
 {
+	[Desc("Replaces the sprite during build up.")]
```

Comment:
"build up" -> "construction".  The german translation for "build up" would be "anhäufen", "ansammeln", so doesn't quite fit.